### ISSUE TITLE
Expand React Hotel coursework with more exercises

### ISSUE THREAD

**Issue by 40thieves** [CodeYourFuture/syllabus#469]
**Which module(s) and week(s) does this change affect?**
Module(s): React
Week(s): All

**What is the work that needs to be done?**

Add some more exercises to the React [Hotel coursework](https://github.com/CodeYourFuture/cyf-hotel-react).

These exercises should allow the trainees to further practice the concepts taught in the class. They should have fairly "prescriptive" instructions, following the style of the existing exercises.

**Why is this work important to do?**

Based on experience of the last couple of London classes, a lot of trainees complete the Hotel coursework quite quickly (~1 - 2 hours). I think it would be beneficial to have more exercises to practice by guiding to the correct answer (due to their "prescriptive" nature).

In contrast, we also set the [React challenges](https://github.com/CodeYourFuture/cyf-react-challenges) as coursework, and these can take up a significant amount of trainee's time. However the style of these exercises is very different, with almost no guidance given to trainees.

In other words, I think we should balance the amount of guidance provided on the React coursework may help struggling trainees, while still leaving open more challenging tasks for stronger trainees. Overall the goal would be to "smooth the slope" from lesson (["I do" / "We do"](https://www.evidencebasedteaching.org.au/the-i-do-we-do-you-do-model-explained/)) to challenges ("You do").

**Additional context**

It is worth noting that we need to migrate the coursework exercises to a 4 week format (https://github.com/CodeYourFuture/syllabus/issues/468), so we should probably do this at the same time.

**Who might need to know about this change?**

@codeyourfuture/react
